lib/logstash/plugin_mixins/http_client.rb

```diff
@@ -53,6 +58,11 @@ def setup_http_client_config
     # 2. Proxy host in form: {host => "proxy.org", port => 80, scheme => 'http', user => 'username@host', password => 'password'}
     # 3. Proxy host in form: {url =>  'http://proxy.org:1234', user => 'username@host', password => 'password'}
     config :proxy
+
+    # If you'd like to use a client certificate (note, most people don't want this) set the path to the x509 cert here
+    config :client_cert, :validate => :path
+    # If you'd like to use a client certificate (note, most people don't want this) set the path to the x509 key here
```

Comment:
'x509 key' isn't right, I don't think. X509 is a standard for representing certificates, not private keys. Mabye "set the path to the private key" ? Not sure on wording here. This will be the private key generated by something like `openssl genrsa` or similar tooling.